Jenkinsfile

```diff
@@ -1,17 +1,21 @@
 pipeline {
-    agent { label 'rosie' }
+    agent none
 
     stages {
         stage('Build') {
+            agent { label 'rosie' }
             steps {
-                sh 'make docker-rpm'
+                sh 'make clean docker-rpm'
+                stash name: 'rpm', includes: 'dist/*.rpm'
```

Comment:
What do stash and unstash do? If they're relatively uncommon commands then add a comment please. If they're common and I'm just not familiar with jenkinsfiles then just leave it alone